### PR TITLE
Add a Multiselec for choosing Pupils to the "add class" form

### PR DESCRIPTION
**Pull request by Hrabarchuk**

### CODE REVIEW

File: src/app/components/add-class-form/add-class-form.component.html [hunk 1]
```diff
@@ -17,5 +17,11 @@ <h1 class="title">add class</h1>
           <mat-option *ngFor="let teacher of teachers" [value]="teacher.id">{{ teacher.fullName}}</mat-option>
         </mat-select>
     </mat-form-field>
+    <mat-form-field  *ngIf="showPupils">
+            <mat-label>Select a pupils to class</mat-label>
+            <mat-select formControlName="selectedPupils" multiple>
+              <mat-option *ngFor="let pupils of listOfPupils" [value]="pupils.id">{{pupils.fullName}}</mat-option>
+            </mat-select>
+          </mat-form-field>
```
Comment (DmMov) [inline src/app/components/add-class-form/add-class-form.component.html]: Align it

File: src/app/components/add-class-form/add-class-form.component.ts [hunk 2]
```diff
@@ -74,14 +80,30 @@ export class AddClassFormComponent implements OnInit {
     }
   }
 
-  ngOnInit(): void {
+
+  ngOnInit() {
+
     this.getTeachersToSelectData();
```
Comment (DmMov) [inline src/app/components/add-class-form/add-class-form.component.ts]: Too much empty lines

File: src/app/components/add-class-form/add-class-form.component.ts [hunk 3]
```diff
@@ -74,14 +80,30 @@ export class AddClassFormComponent implements OnInit {
...
+   this.form.get('number').valueChanges.subscribe(val => {
+      if(val != ""){
+        this.showPupils = true;
+        console.log(typeof(val));
+        this.getPupilsToSelectData(val);
+      }
+      else{
+        this.showPupils = false;
+      }
+    });
```
Comment (DmMov) [inline src/app/components/add-class-form/add-class-form.component.ts]: I don't understand how all this code is aligned

File: src/app/services/pupils/pupilinfo.service.ts [hunk 4]
```diff
@@ -21,5 +22,9 @@ export class PupilinfoService {
     this.httpOptions.loadHeaders();
     return this.http.get<IPupil>(this.url + "/GetPupilById", this.httpOptions.options);
   }
+  getPupilsToSelect = (numb: number): Observable<IPupilToSelect[]> => {
+    this.httpOptions.loadHeaders();
+    return this.http.get<IPupilToSelect[]>(this.url + "/GetPupilsBySchoolShort" +"?numbOfClass=" + numb , this.httpOptions.options);
```
Comment (DmMov) [inline src/app/services/pupils/pupilinfo.service.ts]: You missed a space and have one extra space.

File: src/app/components/add-class-form/add-class-form.component.ts (hunk 3 above)
Comment (Redingston) [reply src/app/components/add-class-form/add-class-form.component.ts]: Ctrl+A then Ctrl+K+F

**Review by Redingston — APPROVED** [review]

**Review by bodkia22 — APPROVED** [review]

**Review by vunt1k — APPROVED** [review]

**Pull request merged**